lib/webistrano/deployer.rb

```diff
@@ -96,40 +96,37 @@ def save_revision(config)
     rescue => e
       logger.important "Could not save revision: #{e.message}"
     end
-    
+
     # saves the process ID of this running deployment in order
     # to be able to kill it
     def save_pid
       @deployment.pid = Process.pid
       @deployment.save!
     end
-    
+
     # override in order to use DB logger
     def instantiate_configuration #:nodoc:
       config = Webistrano::Configuration.new
       config.logger = logger
       config
     end
-    
-    def set_up_config(config)
-      set_pre_vars(config)
-      load_recipes(config)
 
+    def set_up_config(config)
       set_project_and_stage_names(config)
       set_stage_configuration(config)
       set_stage_roles(config)
-      
-      load_project_template_tasks(config)
+
+      # load_project_template_tasks(config)
```

Comment:
Why is this commented out? If there's a good reason to keep it around, leave a code comment explaining what that good reason is. Otherwise, please delete the code.